keyboards/handwired/consolekeyboard/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+#LTO_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
Does this need enabling when its already in the main rules.mk (have committed your suggestion)

Reply:
Nope, the config files are essentially merged together during compilation as necessary.

Reply:
Ahh i misread (am on mobile) red means delete line doesnt it